spacy/lang/fr/lex_attrs.py

```
@@ -4,33 +4,37 @@
+)
 
-_ordinal_words = set("""
+_ordinal_words = set(
+    """
 premier deuxième second troisième quatrième cinquième sixième septième huitième neuvième dixième
 onzième douzième treizième quatorzième quinzième seizième dix-septième dix-huitième dix-neufième
 vingtième trentième quanrantième cinquantième soixantième septantième quatre-vingtième huitantième nonantième
 centième millième millionnième milliardième billionnième quadrillionnième quintillionnième
 sextillionnième septillionnième octillionnième nonillionnième decillionnième
-""".split())
+""".split()
```

Ruff (0.12.2)

Consider using a list literal instead of `str.split`

Replace with list literal

(SIM905)